opentasks/proguard.cfg

```diff
@@ -77,4 +77,6 @@
     java.lang.String TAG;
     @org.dmfs.android.retentionmagic.annotations.* <fields>;
     private long mId;
-}
\ No newline at end of file
+}
+
+-dontwarn au.com.codeka.carrot.**
```

Comment:
is this still necessary?

Reply:
I was just checking that, no there were no more warnings, so I've removed it.